### PR TITLE
New Module: cloud volume for azure snapshot

### PR DESCRIPTION
**Pull request by carchi8py**
NetApp snapshot for Azure 

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- azure_rm_netapp_snapshot.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@carchi8py Thank you for contributor this PR. Could you fix the CI fail, when you finished changed, I will push for review! Thanks a lot!

**Comment by carchi8py** [comment]
@fred-sun the fix for this is in pr #59939, i'm talking with Matt and the Redhat team on how to handle the unit test. We might just drop them.

**Comment by Fred-sun** [comment]
> @Fred-sun the fix for this is in pr #59939, i'm talking with Matt and the Redhat team on how to handle the unit test. We might just drop them.

Thank you for your attention this PR!

**Comment by Fred-sun** [comment]
@carchi8py Thanks for you update! Do you mean that this PR will be closed and replaced by #59939?

**Comment by anshulbehl** [comment]
@carchi8py This is still using the get_client_from_cli_profile(AzureNetAppFilesManagementClient), please fix accordingly

File: lib/ansible/modules/cloud/azure/azure_rm_netapp_snapshot.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        )
+        self.na_helper = NetAppModule()
+        self.parameters = self.na_helper.set_parameters(self.module.params)
+
+        # authentication - using CLI
+        if HAS_AZURE_MGMT_NETAPP is False:
+            self.module.fail_json(msg="the python Azure-mgmt-NetApp module is required")
+        if HAS_AZURE_COMMON is False:
+            self.module.fail_json(msg="the python azure-common module is required")
+        self.client = get_client_from_cli_profile(AzureNetAppFilesManagementClient)
```
Comment (thedoubl3j) [inline lib/ansible/modules/cloud/azure/azure_rm_netapp_snapshot.py:149]: This needs to be fixed per @anshulbehl comment.

**Review by thedoubl3j — CHANGES_REQUESTED** [review]

**Comment by Fred-sun** [comment]
kindly ping!

**Comment by Fred-sun** [comment]
@carchi8py Another, you should add document and return info for test_azure_rm_netapp_snapshot and azure_rm_netapp_snapshot. Thank you very much!

**Comment by thedoubl3j** [comment]
hey @Fred-sun this work has moved to a collection. you can find the new existing code here https://github.com/ansible/ansible_collections_netapp/tree/master/ansible_collections/netapp/azure/plugins/modules.

**Comment by thedoubl3j** [comment]
Closing as module has moved to outside collection repo linked above.

**Pull request closed without merging**